compiler/rustc_apfloat/src/ppc.rs

```diff
@@ -30,7 +30,7 @@ pub type DoubleDouble = DoubleFloat<ieee::Double>;
 // FIXME: Implement all operations in DoubleDouble, and delete these
 // semantics.
 // FIXME(eddyb) This shouldn't need to be `pub`, it's only used in bounds.
-pub struct FallbackS<F>(F);
+pub struct FallbackS<F>(#[allow(dead_code)] F);
```

Comment:
`rustc_apfloat` fix seems unintrusive enough, and only covers abstractions introduced during the port (not anything that has a direct connection to the original C++ code), so I guess you can consider this me checking off on this change :heavy_check_mark:.